webpack/assets/javascripts/react_app/redux/reducers/index.js

```diff
@@ -4,9 +4,14 @@ import hosts from './hosts';
 import notifications from './notifications/';
 import toasts from './toasts';
 
-export default combineReducers({
-  statistics,
-  hosts,
-  notifications,
-  toasts,
-});
+export function combineReducersAsync(asyncReducers) {
+  return combineReducers({
+    statistics,
```

Comment:
does it make sense to use the new api for all redcuers?

Reply:
Maybe
```js

const staticReducers = { ... };

export const combineReducersAsync = (asyncReducers = {}) =>
    combineReducers({ ...staticReducers, ...asyncReducers });

export default combineReducers(staticReducers);
```

Reply:
I actually like the idea of having a single way that all reducers are registered and taking out the work `async` out everywhere and just call it `addReducer`. That would make it trivial to add a `removeReducer` later on if we ever wanted it.

Reply:
I'll refactor a bit and submit an updated version.

Reply:
Alright I worked through the code some more and I think we should stick with what we have. Although it would be nice to have everything using the same api, it just makes more sense in the code to have static reducers registered, well... statically.
If all reducers were registered via the `addReducer` method we would still need to import all the static reducers but then we'd need to call the `addReducer` method on each one which is not as declarative as what we have now.

@sharvit mentioned moving the static reducers into their own file. I think what we have is a really common practice and the `asyncReducers` variable name implies that the others are static.

Let me know if there's any difference of opinion there.